app/views/__tests__/FeatureFlagToggles.spec.js

```diff
@@ -0,0 +1,46 @@
+import { act, fireEvent, render, wait } from '@testing-library/react-native';
+import React from 'react';
+
+import * as flagsEnv from '../../constants/flagsEnv';
+import * as Flags from '../../helpers/Flags';
+import { FeatureFlagsScreen } from '../FeatureFlagToggles';
+
+const { FlagsProvider } = Flags;
+const buildTimeFlags = { feature1: true };
+
+flagsEnv.buildTimeFlags = buildTimeFlags;
```

Comment:
potential global mutation, + not reset per test = scary

And I wonder if this is related to why the test code does not toggle....